[06/27/18, 22:23:00:005] info: ╔═════════════════════════════════════════════════════════════════════════════════╗
[06/27/18, 22:23:00:009] info: ║      Slack 3.2.1, linux 4.10.0-38-generic on x64 at 06/27/18, 22:22:59:997      ║
[06/27/18, 22:23:00:010] info: ╚═════════════════════════════════════════════════════════════════════════════════╝
[06/27/18, 22:23:00:187] info: Initializing local storage instance at path: /home/nozomi/dotfiles/.config/Slack/local-settings.json
[06/27/18, 22:23:00:781] error: Desktop module calls failed to load
{
  "error": {
    "message": "libCallsCore.so: å±æãªãã¸ã§ã¯ããã¡ã¤ã«ãéãã¾ãã: ãã®ãããªãã¡ã¤ã«ããã£ã¬ã¯ããªã¯ããã¾ãã",
    "stack": "Error: libCallsCore.so: å±æãªãã¸ã§ã¯ããã¡ã¤ã«ãéãã¾ãã: ãã®ãããªãã¡ã¤ã«ããã£ã¬ã¯ããªã¯ããã¾ãã\n    at process.module.(anonymous function) [as dlopen] (ELECTRON_ASAR.js:172:20)\n    at Object.Module._extensions..node (module.js:671:18)\n    at Object.module.(anonymous function) [as .node] (ELECTRON_ASAR.js:186:18)\n    at Module.load (module.js:561:32)\n    at tryModuleLoad (module.js:504:12)\n    at Function.Module._load (module.js:496:3)\n    at Module.require (module.js:586:17)\n    at require (internal/module.js:11:18)\n    at bindings (/usr/lib/slack/resources/app.asar/node_modules/@slack/slack-calls-prebuilt/modules/@slack/slack-calls/node_modules/bindings/bindings.js:81:44)\n    at Object.<anonymous> (/usr/lib/slack/resources/app.asar/node_modules/@slack/slack-calls-prebuilt/modules/@slack/slack-calls/index.js:1:192)"
  }
}
[06/27/18, 22:23:00:781] error: Desktop module calls failed to load
[06/27/18, 22:23:00:807] info: cld3::loadModule: loading cld3 module
[06/27/18, 22:23:00:827] info: cld3Loader::loadModule: running environment is NODE
[06/27/18, 22:23:00:828] info: cld3Loader::loadModule: constructed module object for runtime
[06/27/18, 22:23:01:023] info: cld3Loader::initializeRuntime: successfully initialized module
[06/27/18, 22:23:01:023] info: cld3Loader::loadModule: initialized wasm binary Runtime
[06/27/18, 22:23:01:034] info: SpellChecker:: updateSpellcheckerLanguage: Skip to set new language null, v2 spellchecker is not available
[06/27/18, 22:23:01:045] info: Main window preload script finished
{
  "readyTime": 1528.345377
}
[06/27/18, 22:23:01:960] info: Store: BACKGROUND_CLIENTS_INITIALIZED
[06/27/18, 22:23:02:067] info: Store: SET_FULL_SCREEN
[06/27/18, 22:23:03:113] info: Store: START_BACKGROUND_CLIENT
{
  "teamId": "TB47MRNBZ",
  "token": "xoxs-⋯"
}
[06/27/18, 22:23:03:113] info: Store: BACKGROUND_CLIENTS_INITIALIZED true
[06/27/18, 22:23:03:114] info: Store: SET_FULL_SCREEN
[06/27/18, 22:23:03:114] info: Store: SET_NETWORK_STATUS online
[06/27/18, 22:23:03:565] info: Store: SET_BACKGROUND_CLIENT_FIELDS
{
  "status": "connected",
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:23:03:565] info: Store: BACKGROUND_CLIENT_ERROR
{
  "code": 4001,
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:23:03:567] info: Store: SAVE_WINDOW_SETTINGS
{
  "isMaximized": false,
  "position": [
    575,
    131
  ],
  "size": [
    1024,
    768
  ]
}
[06/27/18, 22:23:03:571] info: Store: UPSERT_WEB_CONTENTS
{
  "id": 4,
  "state": "loaded",
  "teamId": "TB9PVC95H"
}
[06/27/18, 22:23:04:588] info: Skipping request to provide hint content for channel for spellchecker
{
  "version": "V2"
}
[06/27/18, 22:23:04:919] info: Store: UPDATE_SESSION_STATUS
{
  "hasValidSession": true,
  "teamId": "TB9PVC95H"
}
[06/27/18, 22:23:04:950] info: Store: SELECT_CHANNEL DB9T2S1C2
[06/27/18, 22:23:04:950] info: Store: UPDATE_TEAM_LOCALE
{
  "locale": "en-US",
  "teamId": "TB9PVC95H"
}
[06/27/18, 22:23:04:950] info: Store: UPDATE_TEAM_THEME
{
  "teamId": "TB9PVC95H",
  "theme": {
    "active_item": "#F79F66",
    "active_item_text": "#FFFFFF",
    "active_presence": "#F79F66",
    "badge": "#F15340",
    "column_bg": "#0D7E83",
    "hover_item": "#D37C71",
    "menu_bg": "#076570",
    "text_color": "#FFFFFF"
  }
}
[06/27/18, 22:23:05:137] info: Store: SET_FULL_SCREEN
[06/27/18, 22:23:07:434] info: Store: UPSERT_WEB_CONTENTS
{
  "id": 3,
  "state": "loaded",
  "teamId": "T031FNTUT"
}
[06/27/18, 22:23:07:785] info: Store: UPDATE_SESSION_STATUS
{
  "hasValidSession": true,
  "teamId": "T031FNTUT"
}
[06/27/18, 22:23:08:818] info: Store: SELECT_CHANNEL C9Z4BU1C2
[06/27/18, 22:23:08:822] info: Store: UPDATE_TEAM_LOCALE
{
  "locale": "en-US",
  "teamId": "T031FNTUT"
}
[06/27/18, 22:23:08:847] info: Store: UPDATE_TEAM_THEME
{
  "teamId": "T031FNTUT",
  "theme": {
    "active_item": "#95BFEA",
    "active_item_text": "#FFFFFF",
    "active_presence": "#FFFF00",
    "badge": "#AAFFFF",
    "column_bg": "#40E0D0",
    "hover_item": "#48D1CC",
    "menu_bg": "#AAFFFF",
    "text_color": "#FFFFFF"
  }
}
[06/27/18, 22:23:14:231] error: Jun-27 22:23:14.230 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:14:246] error: 2018/6/27 22:23:14.246 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: ASSocket: timed out reading 4 bytes from adminserver-mj95:10443","stack":"Error: Unexpected start error: ASSocket: timed out reading 4 bytes from adminserver-mj95:10443\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:15:951] error: Jun-27 22:23:15.950 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:15:972] error: 2018/6/27 22:23:15.968 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:18:494] info: Store: SET_FULL_SCREEN
[06/27/18, 22:23:20:619] error: Jun-27 22:23:20.618 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:20:628] error: 2018/6/27 22:23:20.628 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:24:870] info: Store: SET_FULL_SCREEN
[06/27/18, 22:23:24:970] error: Jun-27 22:23:24.970 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:24:977] error: 2018/6/27 22:23:24.976 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:28:797] info: Store: SET_FULL_SCREEN
[06/27/18, 22:23:28:924] error: Jun-27 22:23:28.923 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:28:929] error: 2018/6/27 22:23:28.928 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:33:299] info: Store: START_BACKGROUND_CLIENT
{
  "teamId": "TB47MRNBZ",
  "token": "xoxs-⋯"
}
[06/27/18, 22:23:33:716] info: Store: SET_BACKGROUND_CLIENT_FIELDS
{
  "status": "connected",
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:23:33:725] info: Store: BACKGROUND_CLIENT_ERROR
{
  "code": 4001,
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:23:34:998] error: Jun-27 22:23:34.998 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:35:003] error: 2018/6/27 22:23:35.300 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:47:927] error: Jun-27 22:23:47.927 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:47:945] error: 2018/6/27 22:23:47.944 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:23:48:909] info: Store: SET_FULL_SCREEN
[06/27/18, 22:23:59:904] error: Jun-27 22:23:59.903 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:23:59:919] error: 2018/6/27 22:23:59.918 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:24:03:746] info: Store: START_BACKGROUND_CLIENT
{
  "teamId": "TB47MRNBZ",
  "token": "xoxs-⋯"
}
[06/27/18, 22:24:04:164] info: Store: SET_BACKGROUND_CLIENT_FIELDS
{
  "status": "connected",
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:24:04:942] info: Store: SET_FULL_SCREEN
[06/27/18, 22:24:10:974] info: Store: SET_FULL_SCREEN
[06/27/18, 22:24:14:473] info: Store: BACKGROUND_CLIENT_ERROR
{
  "code": 4337,
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:24:14:495] info: Store: UPSERT_WEB_CONTENTS
{
  "id": 5,
  "state": "unloaded",
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:24:14:581] info: Store: SET_FULL_SCREEN
[06/27/18, 22:24:17:355] info: Store: SET_FULL_SCREEN
[06/27/18, 22:24:18:317] info: Store: SAVE_WINDOW_SETTINGS
{
  "isMaximized": false,
  "position": [
    574,
    106
  ],
  "size": [
    1024,
    768
  ]
}
[06/27/18, 22:24:20:361] info: Store: UPSERT_WEB_CONTENTS
{
  "id": 5,
  "state": "loaded",
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:24:20:772] error: Jun-27 22:24:20.772 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:24:20:775] error: 2018/6/27 22:24:20.775 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:24:20:913] info: Store: UPDATE_SESSION_STATUS
{
  "hasValidSession": true,
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:24:21:563] info: Store: SELECT_CHANNEL CB47MRRTR
[06/27/18, 22:24:21:567] info: Store: UPDATE_TEAM_LOCALE
{
  "locale": "en-US",
  "teamId": "TB47MRNBZ"
}
[06/27/18, 22:24:21:580] info: Store: UPDATE_TEAM_THEME
{
  "teamId": "TB47MRNBZ",
  "theme": {
    "active_item": "#6698C8",
    "active_item_text": "#FFFFFF",
    "active_presence": "#94E864",
    "badge": "#78AF8F",
    "column_bg": "#303E4D",
    "hover_item": "#4A5664",
    "menu_bg": "#2C3849",
    "text_color": "#FFFFFF"
  }
}
[06/27/18, 22:24:26:902] error: Jun-27 22:24:26.901 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:24:26:908] error: 2018/6/27 22:24:26.907 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:24:33:218] error: Jun-27 22:24:33.217 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:24:33:228] error: 2018/6/27 22:24:33.227 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:24:40:920] error: Jun-27 22:24:40.920 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:24:40:923] error: 2018/6/27 22:24:40.922 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:24:48:899] error: Jun-27 22:24:48.899 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:24:48:902] error: 2018/6/27 22:24:48.902 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:25:06:912] error: Jun-27 22:25:06.912 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:25:06:915] error: 2018/6/27 22:25:06.915 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:25:44:962] error: Jun-27 22:25:44.961 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:25:44:972] error: 2018/6/27 22:25:44.971 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:26:44:031] error: Jun-27 22:26:44.030 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:26:44:039] error: 2018/6/27 22:26:44.380 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:27:46:039] error: Jun-27 22:27:46.039 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:27:46:046] error: 2018/6/27 22:27:46.450 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:28:48:460] error: Jun-27 22:28:48.460 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:28:48:469] error: 2018/6/27 22:28:48.468 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:29:24:380] error: Jun-27 22:29:24.379 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:29:24:384] error: 2018/6/27 22:29:24.384 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:30:26:000] error: Jun-27 22:30:26.000 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:30:26:007] error: 2018/6/27 22:30:26.600 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:31:27:981] error: Jun-27 22:31:27.980 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:31:27:988] error: 2018/6/27 22:31:27.988 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:32:29:960] error: Jun-27 22:32:29.959 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:32:29:968] error: 2018/6/27 22:32:29.968 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:33:32:070] error: Jun-27 22:33:32.070 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:33:32:074] error: 2018/6/27 22:33:32.730 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:34:34:039] error: Jun-27 22:34:34.038 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:34:34:047] error: 2018/6/27 22:34:34.470 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:34:42:929] error: Jun-27 22:34:42.928 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:34:42:939] error: 2018/6/27 22:34:42.939 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:34:48:923] error: Jun-27 22:34:48.923 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:34:48:925] error: 2018/6/27 22:34:48.925 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:34:52:938] error: Jun-27 22:34:52.938 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:34:52:945] error: 2018/6/27 22:34:52.945 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:34:56:921] error: Jun-27 22:34:56.921 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:34:56:930] error: 2018/6/27 22:34:56.929 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:35:01:912] error: Jun-27 22:35:01.910 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:35:01:922] error: 2018/6/27 22:35:01.922 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:35:10:909] error: Jun-27 22:35:10.909 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:35:10:912] error: 2018/6/27 22:35:10.912 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:35:15:942] error: Jun-27 22:35:15.942 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:35:15:953] error: 2018/6/27 22:35:15.952 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:35:24:931] error: Jun-27 22:35:24.930 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:35:24:939] error: 2018/6/27 22:35:24.939 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:35:33:906] error: Jun-27 22:35:33.906 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:35:33:909] error: 2018/6/27 22:35:33.909 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:35:47:010] error: Jun-27 22:35:47.009 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:35:47:021] error: 2018/6/27 22:35:47.200 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:36:22:010] error: Jun-27 22:36:22.010 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:36:22:018] error: 2018/6/27 22:36:22.180 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:37:10:989] error: Jun-27 22:37:10.988 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:37:11:001] error: 2018/6/27 22:37:11.100 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:37:49:060] error: Jun-27 22:37:49.060 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:37:49:062] error: 2018/6/27 22:37:49.620 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:38:11:933] error: Jun-27 22:38:11.932 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:38:11:935] error: 2018/6/27 22:38:11.934 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:39:04:993] error: Jun-27 22:39:04.992 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:39:04:996] error: 2018/6/27 22:39:04.996 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:40:07:020] error: Jun-27 22:40:07.019 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:40:07:026] error: 2018/6/27 22:40:07.250 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:41:09:090] error: Jun-27 22:41:09.090 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:41:09:097] error: 2018/6/27 22:41:09.960 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:41:37:946] error: Jun-27 22:41:37.946 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:41:37:955] error: 2018/6/27 22:41:37.955 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:42:28:000] error: Jun-27 22:42:27.999 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:42:28:009] error: 2018/6/27 22:42:28.800 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:43:30:024] error: Jun-27 22:43:30.023 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:43:30:028] error: 2018/6/27 22:43:30.280 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:44:31:964] error: Jun-27 22:44:31.963 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:44:31:967] error: 2018/6/27 22:44:31.966 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:45:21:963] error: Jun-27 22:45:21.963 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:45:21:971] error: 2018/6/27 22:45:21.970 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:46:24:004] error: Jun-27 22:46:24.004 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:46:24:016] error: 2018/6/27 22:46:24.160 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:47:26:004] error: Jun-27 22:47:26.004 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:47:26:012] error: 2018/6/27 22:47:26.110 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:48:38:253] error: Jun-27 22:48:38.252 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:48:38:261] error: 2018/6/27 22:48:38.261 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: ASSocket: timed out reading 4 bytes from adminserver-hdsq:10443","stack":"Error: Unexpected start error: ASSocket: timed out reading 4 bytes from adminserver-hdsq:10443\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:49:40:039] error: Jun-27 22:49:40.039 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:49:40:048] error: 2018/6/27 22:49:40.480 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:50:41:958] error: Jun-27 22:50:41.958 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:50:41:960] error: 2018/6/27 22:50:41.960 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:51:36:968] error: Jun-27 22:51:36.967 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:51:36:980] error: 2018/6/27 22:51:36.979 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:52:38:963] error: Jun-27 22:52:38.963 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:52:38:970] error: 2018/6/27 22:52:38.970 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:53:41:022] error: Jun-27 22:53:41.022 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:53:41:040] error: 2018/6/27 22:53:41.390 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:54:42:979] error: Jun-27 22:54:42.978 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:54:42:982] error: 2018/6/27 22:54:42.982 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:55:31:722] info: Store: FOREGROUND_APP
[06/27/18, 22:55:31:747] info: Store: SET_FULL_SCREEN
[06/27/18, 22:55:31:749] info: Store: SET_FULL_SCREEN
[06/27/18, 22:55:31:774] info: Store: SET_FULL_SCREEN
[06/27/18, 22:55:32:747] info: Store: SAVE_WINDOW_SETTINGS
{
  "isMaximized": false,
  "position": [
    575,
    131
  ],
  "size": [
    1024,
    768
  ]
}
[06/27/18, 22:55:33:750] info: Store: MAIN_WINDOW_FOCUSED
[06/27/18, 22:55:34:364] info: Store: MAIN_WINDOW_FOCUSED
[06/27/18, 22:55:34:547] info: Store: SELECT_TEAM TB47MRNBZ
[06/27/18, 22:55:41:722] info: Store: SET_FULL_SCREEN
[06/27/18, 22:55:42:709] info: Store: SAVE_WINDOW_SETTINGS
{
  "isMaximized": false,
  "position": [
    574,
    106
  ],
  "size": [
    1024,
    768
  ]
}
[06/27/18, 22:55:45:019] error: Jun-27 22:55:45.019 SOCKET-MANAGER (TB9PVC95H) Closing socket because #4106: received an "error" message from socket
[06/27/18, 22:55:45:024] error: 2018/6/27 22:55:45.230 logging error: {"subtype":"flannel_start_error","message":"Unexpected start error: Too many connections","stack":"Error: Unexpected start error: Too many connections\n    at e (https://cfr.slack-edge.com/bv1-4/rollup-core_required_ts.2e3e449bfeaa7d40c5b8.min.js:1:21350)\n    at t.execute (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2658239)\n    at o.dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2660661)\n    at dispatch (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2659038)\n    at https://cfr.slack-edge.com/bv1-4/modern.vendor.fa67e26714c1d85c5a78.min.js:1:860\n    at c (https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:361547)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102817\n    at Array.forEach (<anonymous>)\n    at https://cfr.slack-edge.com/bv1-4/application.cf0badad773cd189735c.min.js:1:1102586\n    at i (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:2005699)\n    at S._settlePromiseFromHandler (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1978184)\n    at S._settlePromise (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979036)\n    at S._settlePromiseCtx (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1979576)\n    at i._drainQueue (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936750)\n    at i._drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1936803)\n    at drainQueues (https://cfr.slack-edge.com/bv1-4/rollup-core_required_libs.8df12659b2eea0c98c5e.min.js:1:1935093)"}
[06/27/18, 22:55:47:449] info: Store: FOREGROUND_APP
[06/27/18, 22:55:47:500] info: Store: SET_FULL_SCREEN
[06/27/18, 22:55:47:507] info: Store: SET_FULL_SCREEN
[06/27/18, 22:55:47:518] info: Store: SET_FULL_SCREEN
[06/27/18, 22:55:47:524] info: Store: MAIN_WINDOW_FOCUSED
[06/27/18, 22:55:48:488] info: Store: SAVE_WINDOW_SETTINGS
{
  "isMaximized": false,
  "position": [
    575,
    131
  ],
  "size": [
    1024,
    768
  ]
}
[06/27/18, 22:55:50:244] info: Store: QUIT_APP
[06/27/18, 22:55:50:259] info: Store: UPDATE_TEAM_USAGE
{
  "T031FNTUT": 21539470,
  "TB47MRNBZ": 15691
}
[06/27/18, 22:55:50:298] info: Store: REMOVE_WINDOW 2
[06/27/18, 22:55:50:309] info: Store: QUIT_APP
[06/27/18, 22:55:50:313] info: Store: SET_FULL_SCREEN
[06/27/18, 22:55:50:358] info: Store: QUIT_APP
